src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+<sky-docs-demo-page
+  pageTitle="Buttons"
+>
+  <sky-docs-demo-page-summary>
+    The button classes create buttons to trigger actions from within an interface. For information about styles and implementation, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/buttons-links">buttons and links guidelines</a>.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Button classes"
+    >
+      <table class="sky-docs-table">
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-primary">Primary</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              The primary button triggers the most important or most frequently used action on a page or the progressive terminal action on a modal (such as <strong>Save</strong> or <strong>Done</strong>). Pages and modals should only include one primary button to ensure that it draws user attention. For information about how to use primary actions, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/page-layouts#primary-action-guidelines">primary action guidelines</a>.
```

Comment:
I'd delete `or Done` in the parentheses. A question came up on this recently asking why we were using **Done** instead of **Save**, and there was no clear guidance on the distinction between the two terminal actions, so we opted for **Save**. We didn't forbit **Done**, but it seemed like the consensus was to stick with a common term instead of using synonyms, so I wouldn't include synonyms as examples in our guidance.